documentation.md

```diff
@@ -23,10 +23,10 @@ Brian
 
 Goss
 
-  * AR Method: 
-  * Description:
-  * Example:
-  * Anything else?:
+  * AR Method: Select  
+  * Description: Select allows us to return specific columns from a table. It can accept symbols as arguments, an it will return columns with the same name as the symbols. Alternatively, you can pass a string which will be embeded in a SQL Select statement. This is useful for doing calculations or manipulation on each row returned by the select. Select is usually the second method in a chain of Active Record methods, after the table or collection and before methods that transform or limit the result.  
+  * Example: Students.select(:name, :email).order(:name); Teachers.select('average(Time.now - hire_date) as seniority');  

```

Comment:
@bcgoss what's the return value of these queries? what time of objects? could you add some clarification around this?